packages/vuetify/src/components/VSwitch/__tests__/VSwitch.spec.cy.tsx

```diff
@@ -0,0 +1,72 @@
+/// <reference types="../../../../types/cypress" />
+
+import type { CSSProperties } from 'vue'
+import { VSwitch } from '../VSwitch'
+
+describe('VSwitch', () => {
+  describe('color', () => {
+    const parentColor = 'rgb(5, 5, 5)'
+    const vThemeSurfaceCSSVar = 'rgb(255, 255, 255)'
+    const styleContext = {
+      '--v-theme-surface': vThemeSurfaceCSSVar,
+      color: parentColor,
+    }
+
+    it('supports currentColor behaviour when no color props is provided', () => {
+      const colorProp = undefined
+      expectColorPropConversionToRGBColor(colorProp, styleContext.color, styleContext)
+    })
+
+    it('supports RGB custom colors provided as prop', () => {
+      const rgb = 'rgb(255, 0, 242)'
+      expectColorPropConversionToRGBColor(rgb, rgb, styleContext)
+    })
+    it('supports RGBA custom colors provided as prop', () => {
+      const rgba = 'rgba(255, 0, 242, 1)'
+      const rgb = 'rgb(255, 0, 242)'
+
+      expectColorPropConversionToRGBColor(rgba, rgb, styleContext)
+    })
+    it('supports HEX custom colors provided as prop', () => {
+      const hex = '#ff00f2'
+      const rgb = 'rgb(255, 0, 242)'
+
+      expectColorPropConversionToRGBColor(hex, rgb, styleContext)
+    })
+    it('supports HSLA custom colors provided as prop', () => {
+      const hsla = 'hsla(303, 100%, 50%, 1)'
+      const rgb = 'rgb(255, 0, 242)'
+
+      expectColorPropConversionToRGBColor(hsla, rgb, styleContext)
+    })
+    it('supports HSL custom colors provided as prop', () => {
+      const hsl = 'hsla(303, 100%, 50%)'
+      const rgb = 'rgb(255, 0, 242)'
+
+      expectColorPropConversionToRGBColor(hsl, rgb, styleContext)
+    })
+  })
+})
+
+function expectColorPropConversionToRGBColor (
```

Comment:
I would have expected to be able to use `it.each` for this run, but it seems that the Cypress test runner doesn't support that? TypeScript tells me should be using Mocha, which supports it.

So that's strange, I tried googling around but considering the Cy Component Testing is something quite recent, I could not find any useful answer.

If there is a way to run a suite iteratively over a sample array, that'd be better for code maintenance and semantics.

Reply:
- looked at it, doesn't seem supported yet by cypress test runner -